_studio/mfx_lib/decode/h264/src/mfx_h264_dec_decode.cpp

```diff
@@ -967,6 +970,12 @@ mfxStatus VideoDECODEH264::QueryIOSurfInternal(eMFXPlatform platform, eMFXHWType
         dpbSize = par->mfx.MaxDecFrameBuffering;
 
     mfxU32 numMin = dpbSize + 1 + asyncDepth;
+
+#ifdef MFX_ENABLE_CPLIB
```

Comment:
Do we actually need to hide all that under MFX_ENABLE_CPLIB? I feel we can limit MFX_ENABLE_CPLIB usage to just few places where we really create some classes to handle protected content and maybe some checks in Init to return error if anyone tries to use protected if support is not available.

Reply:
fixed